app/assets/javascripts/search.coffee

```diff
@@ -42,3 +42,17 @@ $(document).ready ->
 # Handle presence of Javascript by turning off or on visibility of JS sensitive objects
 $('.js-disabled-object').css 'visibility': 'hidden'
 $('.js-enabled-object').css 'visibility': 'visible'
+
+# Change focus of cursor when search link is clicked on Manage Collections, Manage Variables, or Manage Services pages.
+$ ->
+ $('#search-focus').on 'click', ->
+    document.getElementById('keyword').focus()
```

Comment:
I think what @macrouch may have meant (correct me if I'm wrong) was that you can use `$('#keyword')` here instead of `document.getElementById('keyword')`